files/ko/web/javascript/reference/global_objects/internalerror/index.html

```diff
@@ -1,121 +1,92 @@
 ---
 title: InternalError
 slug: Web/JavaScript/Reference/Global_Objects/InternalError
+tags:
+  - Class
+  - InternalError
+  - JavaScript
+  - Object
 translation_of: Web/JavaScript/Reference/Global_Objects/InternalError
 browser-compat: javascript.builtins.InternalError
 ---
 <div>{{JSRef}} {{non-standard_header}}</div>
 
-<p><strong><code>InternalError </code></strong>객체는 JavaScript 엔진 내부에서 발생한 에러를 나타냅니다. 예를 들면, <strong>"InternalError</strong>: too much recursion"라는 에러가 있으며, 이것은 "내부 에러: 너무 많은 재귀 호출" 상황이 발생했을 때 나타납니다. </p>
+<p><strong><code>InternalError</code></strong> 객체는 JavaScript 엔진 내부에서 발생한 오류를 나타냅니다. </p>
 
-<h2 id="구문">구문</h2>
+<p>다음의 오류 예시의 경우는 일반적으로 어떤 값이 너무 큰 경우입니다.</p>
 
-<pre class="syntaxbox">new InternalError([<var>message</var>[, <var>fileName</var>[, <var>lineNumber</var>]]])</pre>
+<ul>
+ <li>"too many switch cases", (swich case의 수가 너무 많음)</li>
+ <li>"too many parentheses in regular expression", (정규표현식에 너무 많은 괄호가 있음)</li>
+ <li>"array initializer too large", (배열 초기화 값이 너무 큼)</li>
+ <li>"too much recursion". (너무 많은 재귀 호출)</li>
+</ul>
 
-<h3 id="파라메터">파라메터</h3>
+<h2 id="Constructor">생성자</h2>
 
 <dl>
- <dt><code>message</code></dt>
- <dd>선택적 파라메터. 에러에 대한 설명.</dd>
- <dt><code>fileName</code> {{non-standard_inline}}</dt>
- <dd>선택적 파라메터. 예외(Exception)가 발생한 코드를 포함하고 있는 파일 명.</dd>
- <dt><code>lineNumber</code> {{non-standard_inline}}</dt>
- <dd>선택적 파라메터. 예외(Exception)이 발생한 코드의 라인 넘버.</dd>
+ <dt>{{jsxref("InternalError/InternalError", "InternalError()")}}</dt>
+ <dd>새로운 <code>InternalError</code> 객체를 만듭니다.</dd>
 </dl>
 
-<h2 id="설명">설명</h2>
-
-<p><code>InternalError</code> 는 JavaScript 엔진에서 내부 에러가 발생할 때마다 던져집니다.</p>
-
-<p>다음의 에러 예시 들의 경우는 일반적으로 어떤 값이 너무 큰 경우 입니다.</p>
-
-<ul>
- <li>"too many switch cases", (swich case의 수가 너무 많음.)</li>
- <li>"too many parentheses in regular expression", (정규표현식에 너무 많은 괄호가 있음.)</li>
- <li>"array initializer too large", (배열 초기화 값이 너무 큼.)</li>
- <li>"too much recursion". (너무 많은 재귀 호출.)</li>
-</ul>
-
-<h2 id="속성">속성</h2>
+<h2 id="Instance_properties">인스턴스 속성</h2>
 
 <dl>
- <dt>{{jsxref("InternalError.prototype")}}</dt>
- <dd><code>InternalError</code> 객체에 속성을 추가하도록 해줍니다.</dd>
+  <dt>{{jsxref("Error.prototype.message", "InternalError.prototype.message")}}</dt>
+  <dd>오류 메시지.{{jsxref("Error")}}로부터 상속되었습니다.</dd>
+  <dt>{{jsxref("Error.prototype.name", "InternalError.prototype.name")}}</dt>
+  <dd>오류 이름. {{jsxref("Error")}}로부터 상속되었습니다.</dd>
+  <dt>{{jsxref("Error.prototype.fileName", "InternalError.prototype.fileName")}}</dt>
+  <dd>오류가 발생한 파일 경로. {{jsxref("Error")}}로부터 상속되었습니다.</dd>
+  <dt>{{jsxref("Error.prototype.lineNumber", "InternalError.prototype.lineNumber")}}</dt>
+  <dd>오류가 발생한 곳의 줄 위치. {{jsxref("Error")}}로부터 상속되었습니다.</dd>
```

Comment:
```suggestion
  <dd>오류가 발생한 곳의 줄 번호. {{jsxref("Error")}}로부터 상속되었습니다.</dd>
```
> Line number in file that raised this error. Inherited from Error.

`InternalError.prototype.lineNumber`, lineNumber인 만큼 줄 위치보다는 줄 번호가 더 자연스러운 번역으로 느껴져서 리뷰했습니다.